examples/tts/tts.cpp

```diff
@@ -371,7 +371,7 @@ static std::string replace_numbers_with_words(const std::string & input_text) {
 }
 
 // Based on: https://github.com/edwko/OuteTTS/blob/a613e79c489d8256dd657ea9168d78de75895d82/outetts/version/v1/prompt_processor.py#L39
-static std::string process_text(const std::string & text) {
+static std::string process_text(const std::string & text, bool is_version_0_3) {
```

Comment:
btw to check if the version is 0.3, you can use:

```cpp
bool is_version_0_3 = common_get_builtin_chat_template(model) == "outetts-0.3"
```

@edwko I planned to add this as a dedicated GGUF meta key, but turns out I still not have the time to implement this. I'll try to do this in next week! And btw congrats for the release of v0.3 😄